Source/EngineClassesTests/PlayerTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EngineClasses;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace EngineClasses.Tests
+{
+    [TestClass()]
+    public class PlayerTests
+    {
+        [TestMethod()]
+        public void SelectGamePieceTest_PickOutTheGamePieceYouWant_ReturnGamePieceBasedOnIndex()
+        {
+            string userName = "test";
+            Player p1 = new Player(userName, "Red");
+            p1.GamePiece.Add(new GamePiece(true, false));
+            p1.GamePiece.Add(new GamePiece(true, false));
+            p1.GamePiece.Add(new GamePiece(false, false));
+            p1.GamePiece.Add(new GamePiece(true, false));
+
+
+            Assert.IsFalse(p1.SelectGamePiece(2).IsAtBase);
+        }
+
+        [TestMethod]
+        public void RollDiceTest_RollDiceRandomNumber_ReturnOneToSix()
+        {
+            Player player = new Player("TestMan", "Red");
+            List<int> results = new List<int>();
+
+            for (int i = 0; i < 100; i++)
+            {
+                results.Add(player.RollDice());
+            }
+
+            Assert.IsFalse(results.Any(r => r > 6 || r < 1));
```

Comment:
What does this test test? If the Random-class is working? Can there happen anything special at loop 89?

Reply:
Tanken var att slumpa fram ett arbiträrt men tillräckligt stort antal siffror för tat kan se till att inte någon siffra i listan är utanför det tillåtna intervallet. Är nog mer ett test av att vi inte använt random fel än av själva randomklassen. Men det är nog överflödigt, mer kanske ett resultat av att vi velat ha minst ett test för varje metod såvida det inte är helt uppåt väggarna tokigt att testa metoden. Men i detta fallet är det nog som sagt överflödigt!